src/lib/Scenes/Home/Components/Sales/Components/SaleListItem.tsx

```diff
@@ -137,9 +137,8 @@ export default createFragmentContainer(SaleListItem, {
       registration_ends_at
       live_start_at
       cover_image {
-        cropped(width: 158, height: 196, version: "medium") {
-          url
-        }
+        url(version: "large")
+        aspect_ratio
```

Comment:
@l2succes @damassi Don’t use MP’s cropping feature, but instead get a high-res URL and let our image component determine the right resolution and request that from Gemini.